Makefile

```diff
@@ -1,28 +1,31 @@
-.PHONY: build-local build-noop-dest run-local run-latest run-latest-nightly print-results
+BENCHMARKS_PATH := ./benchmarks
 
-build:
-	go build main.go
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+%?:.*?## .*$$' $(MAKEFILE_LIST) | sed 's/^*://g' | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
 
-scripts/benchmark:
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o scripts/benchmark main.go
+.PHONY: install-tools
+install-tools: install-benchi install-csvtk ## Install all tools required for benchmarking.
 
-# Builds a fresh Docker image, so we're not limited on the GA and nightly builds
-build-local:
-	@cd ../conduit && docker build -t conduit:local .
+.PHONY: install-benchi
+install-benchi: ## Install latest version of benchi.
+	# TODO check if already installed
+	curl https://raw.githubusercontent.com/ConduitIO/benchi/main/install.sh | sh
 
-plugins/noop-dest:
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o plugins/noop-dest noopdest/main.go
+.PHONY: install-csvtk
+install-csvtk: ## Install csvtk for processing CSV files.
+	# TODO
```

Comment:
Did you plan to work on this as part of this PR? If not, maybe we can just remove it.